cmd/vic-machine/configure/configure.go

```diff
@@ -405,8 +387,19 @@ func (c *Configure) Run(clic *cli.Context) (err error) {
 		op.Error("Configuring cannot continue: copying configuration failed")
 		return err
 	}
+	// Copy original and merged resources
+	// This copy is needed so that we validate only the resource settings
+	// (--mem, --cpu, etc) supplied by the user.
+	inputResources := c.Data.ResourceLimits
+	mergedResources := oldData.ResourceLimits
+
+	// overwriting user input w/merged dataset
 	c.Data = oldData
 
+	// Set the ResourceLimits to the input received from
+	// the user
+	oldData.ResourceLimits = inputResources
```

Comment:
The combined resource config set is user input and existing data...doesn't really make sense to validate the existing config...we just retrieved it.  Also the validate code for DRS Disabled warn's when you pass a VCH Pool modification -- specifically that it will be ignored since there are no pools in a DRS disabled environment.  We only want to warn for user provided values and not the existing configuration.

Reply:
Slightly off-topic: I have a vaguely similar issue in #7797 where validating _just_ the merged configuration doesn't give the desired result (since at that point we can no longer tell whether an existing entity was something _we_ created or whether it's something that a vSphere user created that we don't want to clobber).

I suspect at some point we're going to need to overhaul configure to operate on `(old, delta, new)` instead of assuming that it can just merge everything up-front and operate on that.

Reply:
Also off-topic, but I'm leaving it here so that I can find it again: We seem to have another case where operating on just the merged configuration doesn't work well and work around it in a (different) one-off way:

https://github.com/vmware/vic/blob/7fd77b0f05a9c3227d461f983261784e6cc6f381/lib/install/management/configure.go#L168-L177